soc/openhwgroup/cva6/Kconfig.defconfig

```
@@ -0,0 +1,27 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_OPENHWGROUP_CVA6
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 25000000
+
+config RISCV_SOC_INTERRUPT_INIT
+	default y
+
+config RISCV_GP
+	default y
+
+config 2ND_LVL_ISR_TBL_OFFSET
+	default 13
+
+config 2ND_LVL_INTR_00_OFFSET
+	default 11
+
+config MAX_IRQ_PER_AGGREGATOR
+	default 186
+
+config NUM_IRQS
+	default 186
```

```suggestion
	default 199
```

We have one aggregator of at most 186 IRQs + an offset of 13, which gives `1 * 186 + 13 = 199`.

---

I have double-checked the hardware, the singular PLIC is actually limited to 32 sources. So I have corrected `MAX_IRQ_PER_AGGREGATOR` to 30 and `NUM_IRQS` to 43.
By the way, cva6 does not currently have any local interrupts (except timer/ipi/software/external connected to PLIC). Are configs `2ND_LVL_ISR_TBL_OFFSET` and  `2ND_LVL_INTR_00_OFFSET` even required in this case, or should I just use their respective defaults?

---

I will disable first-level interrupt support then; I have checked that the standard first-level interrupts are handled by the RISC-V ISR.

---

Nevermind. The PLIC driver needs to register itself as handler for a first-level interrupt (11, machine external), making it necessary to support first and second levels.